app/assets/stylesheets/active_admin_sidebar.scss

```diff
@@ -0,0 +1,88 @@
+@import "active_admin_sidebar_pure_icons";
+
+body.active_admin #active_admin_content.left_sidebar, body.active_admin #active_admin_content.collapsed_sidebar {
+
+  #sidebar{
+    margin-left:0;
+    display: block;
+  }
+
+  .paginated_collection_contents{
+    clear: none;
+    float: left;
+    width: 100%;
+  }
+
+  .blank_slate_container{
+    clear:none;
+  }
+
+  .columns {
+     clear: none;
+   }  
+
+  #main_content_wrapper{
+    float:inherit;
+    width:auto;
+    margin-left: 298px;
+    #main_content{
+       margin:0;
+       float: inherit;
+       .tabs .comments {
+         .active_admin_comment {
+           clear: none;
+         }
+       }
+     }
+
+  }
+
+  .table_tools:after {
+    clear: none;
+    padding-bottom: 16px;
+  }
+
+  .collapse_btn, .uncollapse_btn {
+    cursor: pointer;
+    background-color: #767270;
+    border-radius: 5px;
+    color: #ffffff;
+  }
+
+  .collapse_btn {
+    display: block;
+    float: right;
+    clear: both;
+  }
+
+  .uncollapse_btn {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Selector `uncollapse_btn` should be written in lowercase with hyphens